### PR TITLE
reduce font wallet address input

### PR DESCRIPTION
**Pull request by hard75**

### CODE REVIEW

File: src/styles/_exchange-form.scss [hunk 1]
```diff
@@ -191,3 +191,20 @@
...
+    font-size: 15px;
+  }
+}
+
+.amount-input {
+  font-size: 15px;
+}
+
+.wallet-address-input {
+  font-size: 11px;
```
Comment (josedahlquist) [inline src/styles/_exchange-form.scss:206]: why is the size of the address smaller than the account and amount? The three of them should have the same size, I even suggest using the same class for this

File: src/styles/_exchange-form.scss [hunk 2]
```diff
@@ -191,3 +191,20 @@
...
+  .v-input__slot {
+    padding: 0 5px !important;
+  }
+}
```
Comment (josedahlquist) [inline src/styles/_exchange-form.scss:210]: Recall leaving an empty line at the end of your files
```suggestion
}

```

File: src/styles/_exchange-form.scss (hunk 1 above)
Comment (hard75) [reply src/styles/_exchange-form.scss:206]: Because wallet address haven't space in to the label and the others labels see very small with 11px

**Review by leoiovlabs — APPROVED** [review]
LGTM

File: src/styles/_exchange-form.scss (hunk 2 above)
Comment (jeremy-then) [inline src/styles/_exchange-form.scss:210]: What about an extra space at the end of this file?

Comment (jeremy-then) [inline src/styles/_exchange-form.scss:210]: What about an extra space at the end of this file?

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Pull request merged**